src/main/vcpf4/usbd_desc.c

```diff
@@ -66,13 +66,8 @@
 #define USBD_LANGID_STRING              0x409
 #define USBD_MANUFACTURER_STRING        FC_FIRMWARE_NAME
 
-#ifdef USBD_PRODUCT_STRING
-  #define USBD_PRODUCT_HS_STRING          USBD_PRODUCT_STRING
-  #define USBD_PRODUCT_FS_STRING          USBD_PRODUCT_STRING
-#else
-  #define USBD_PRODUCT_HS_STRING          "STM32 Virtual ComPort in HS mode"
-  #define USBD_PRODUCT_FS_STRING          "STM32 Virtual ComPort in FS Mode"
-#endif /* USBD_PRODUCT_STRING */
+#define USBD_PRODUCT_HS_STRING          "STM32 Virtual ComPort in HS mode"
+#define USBD_PRODUCT_FS_STRING          "STM32 Virtual ComPort in FS Mode"
```

Comment:
Why is this broken here and the proper USB product string changed to something cryptic that most users will not understand?

Reply:
I'll back out this change. It's from https://github.com/betaflight/betaflight/pull/10596 which addresses non-unified F4 targets not connecting properly on the MacOS configurator. At least for me.

Reply:
Have you tried with a recent 10.8.0 build of configurator? I tried to reproduce, but all flight controllers are recognised properly for me with recent 10.8.0 on MacOS BigSur.
Anyhow, I think this should be fixed in configurator.